tox.ini

```diff
@@ -150,16 +150,6 @@ commands =
     {[testenv:integration]commands}
     {[testenv]covreport}
 
-[testenv:ferc1_schema]
-description = Verify FERC Form 1 DB schema are compatible for all years.
-skip_install = false
-extras =
-    {[testenv:integration]extras}
-commands =
-    pytest {tty:--color=yes} {posargs} {[testenv]covargs} \
-      --etl-settings src/pudl/package_data/settings/etl_full.yml \
-      test/integration/etl_test.py::test_ferc1_schema
-
```

Comment:
Somehow we removed this test, and didn't remove the corresponding test environment.